test/benchmark_http_client_test.cc

```diff
@@ -26,6 +26,26 @@ using namespace testing;
 
 namespace Nighthawk {
 
+namespace BenchmarkClientTestHelperFunctions {
```

Comment:
please make it anonymous namespace. No name is needed for this namespace. It helps readers to know that this won't be referenced anywhere outside this file.

Reply:
Done.